context/app/markdown/docs/assay/rnaseq.md

```diff
@@ -7,48 +7,52 @@
 
 
 ## Definitions
-RNA sequencing reveals the identities and quantities of transcribed genes in a biological sample ([https://en.wikipedia.org/wiki/RNA-Seq](https://en.wikipedia.org/wiki/RNA-Seq)). Evaluation of the total gene expression profile of a biological sample is referred to as “transcriptomics”. Genes are transcribed in response to some signal(s) that the gene products (proteins) are required. Each transcribed RNA transcript then undergoes processing (addition of a poly-A tail, 5’-capping, exon splicing) to generate messenger RNA (mRNA), which is exported from the nucleus to the cytoplasm for translation into the corresponding encoded protein. The derived sequences of each mRNA transcript are aligned against a reference genome to establish the identity of the corresponding gene. Since each mRNA molecule represents a single gene transcript, the total mRNA count aligning to a gene represents the expression profile for that gene.
+RNA sequencing reveals the identities and quantities of transcribed genes in a biological sample. Evaluation of the total gene expression profile of a biological sample is referred to as “transcriptomics”. Genes are transcribed in response to some signal(s) that the gene products (proteins) are required. Each transcribed RNA transcript then undergoes processing (addition of a poly-A tail, 5’-capping, exon splicing) to generate messenger RNA (mRNA), which is exported from the nucleus to the cytoplasm for translation into the corresponding encoded protein. The derived sequences of each mRNA transcript are aligned against a reference genome to establish the identity of the corresponding gene. Since each mRNA molecule represents a single gene transcript, the total mRNA count aligning to a gene represents the expression profile for that gene.
 
 Gene expression profiling begins with isolation of total RNA from a tissue sample or from individual cells. mRNA is typically purified from total RNA by removal of ribosomal RNA (rRNA). mRNA is further processed for sequencing, as described in a detailed example protocol found in Appendix 1 of this document.
 
-A visual summary is provided below. [https://rpubs.com/ewilkinson_KRISP/479615](https://rpubs.com/ewilkinson_KRISP/479615)
-**![](https://lh5.googleusercontent.com/lucozpAWSmdTG4KDF7I9Wfua5rvVxX7WlSCdHnaOAS-oaS92o-96hN9OdGyzerGUXUaUIuim4l6JKqeuvAPLUFib_seMquWlu0aL-U4K_WSSIPdhdVUKD4aMXBysAMXoGcGbr4ev)**
+A visual summary is provided below.
+![](https://lh5.googleusercontent.com/lucozpAWSmdTG4KDF7I9Wfua5rvVxX7WlSCdHnaOAS-oaS92o-96hN9OdGyzerGUXUaUIuim4l6JKqeuvAPLUFib_seMquWlu0aL-U4K_WSSIPdhdVUKD4aMXBysAMXoGcGbr4ev)
+
+Figure 2: A basic representation of the major steps and considerations in the sequencing of RNA.
+[(Kukurba et al, 2015)](https://doi.org/10.1101/pdb.top084970)
+
```

Comment:
```suggestion
### *Bulk RNAseq*
For questions on bulk RNAseq, contact: [Stephanie Nevins](mailto:snevins@stanford.edu)
```

Reply:
This is already present below. Doesn't need to be repeated.